### DISCUSSION TITLE
Heatmap: configure color scale min and max values by variable

### DISCUSSION THREAD

**Discussion by DomiKr** [grafana/grafana#52704] [Dashboards, Panels & Visualizations]
## Problem
I want to analyze traffic volume using a heatmap panel. Since the traffic volume changes over time, I have to manually set the minimum and maximum value of the heatmap color scheme every time I change the time range. This can be very annoying and time consuming.
## What would you like to be added
Being able to set the minimum and maximum value of the heatmap color scale via a variable/variable dropdown would be great, as it would allow me to quickly and easily change the color scale  according to my data.

**Comment by RobRoyce** [comment]
+1 on this. I have variables min_temp and max_temp that I want to use across many heatmaps in the same dashboard. The issue is that using $min_temp and $max_temp in the heatmap panel and clicking "Save" simply erases the variables.

**Comment by morganchristiansson** [comment]
I was expecting this to work. Now I'm disappointed.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
